react/features/base/jwt/middleware.js

```diff
@@ -5,16 +5,18 @@ import {
     CONFERENCE_LEFT,
     CONFERENCE_WILL_LEAVE,
     SET_ROOM
-} from '../base/conference';
-import { SET_CONFIG } from '../base/config';
-import { SET_LOCATION_URL } from '../base/connection';
-import { LIB_INIT_ERROR } from '../base/lib-jitsi-meet';
+} from '../conference';
+import { SET_CONFIG } from '../config';
+import { SET_LOCATION_URL } from '../connection';
+import { LIB_INIT_ERROR } from '../lib-jitsi-meet';
 import {
     getLocalParticipant,
     getParticipantCount,
-    PARTICIPANT_JOINED
-} from '../base/participants';
-import { MiddlewareRegistry } from '../base/redux';
+    LOCAL_PARTICIPANT_DEFAULT_NAME,
+    PARTICIPANT_JOINED,
```

Comment:
style: capitalized ones go last?

Reply:
is that really a thing ? If yes then maybe we should add that to https://github.com/jitsi/jitsi-meet/blob/master/doc/coding-style.md. All those unspoken rules are really annoying.

Reply:
There was also something about imports grouping, but I have no idea how is it supposed to be

Reply:
@paweldomas, let's please talk on the subject of the coding style when you have the time for a fact-to-face meeting so that we can create specific action times to relieve your annoyance and assign them for execution.